codespell_lib/data/dictionary.txt

```diff
@@ -19140,6 +19325,7 @@ unrestrcited->unrestricted
 unroated->unrotated
 unsable->unusable, usable, unstable,
 unsccessful->unsuccessful
+unschedule->unschedules
```

Comment:
`unschedule` is a valid word. Please remove.

Reply:
Removed